resolwe_bio/tests/processes/test_support_processors.py

```diff
@@ -1015,6 +1024,57 @@ def test_spikein_pairwise(self):
 
         self.assertFileExists(sirv_set3, "report_zip")
 
+    @tag_process("rnaseqc-qc")
```

Comment:
We should test the UCSC input option as well, as this file type handling represents a significant portion of the codebase.